dashboard/app/views/levels/edit.html.haml

```diff
@@ -76,6 +76,8 @@
         %div.resetLevel{:style => "height: 35px; margin: 15px"}
           = label_tag 'reset', 'Reset version history when saving and loading level', {:style => "display: inline; margin-top: 4px"}
           = check_box_tag 'reset', '1', true
+        - if [Pythonlab, Javalab, Weblab2].include?(@level.class)
```

Comment:
Do you think it's worth leaving a comment here to this effect?
> added the button for these 3 lab types because they all use the same start_sources route for editing start code.
I wonder if it makes sense to maintain a list of these three labs for other reasons (upper labs?), but we could address that separately.

Reply:
Added a comment. I think long-term it might make sense to make a wrapper method around `edit_blocks_level_path` specifically for start code editing that will figure out what suffix to add based on the lab type, rather than passing through the suffix we want.